src/entry.js

```diff
@@ -15,6 +15,8 @@ if (typeof Components !== 'undefined') {
   // See: https://bugzilla.mozilla.org/show_bug.cgi?id=1007318
   Blob = hiddenWindow.Blob;
   WebSocket = hiddenWindow.WebSocket;
+  FileReader = hiddenWindow.FileReader;
```

Comment:
Sure. I updated `spec.js` and added a binary test to this pull request:
https://github.com/freedomjs/freedom/pull/203